src/FixedProductMarketMakerMapping.ts

```diff
@@ -351,6 +358,8 @@ export function handleSell(event: FPMMSell): void {
   );
   fpmm.save();
 
+  updateGlobalOpenInterest(returnAmountPlusFees, TRADE_TYPE_SELL);
```

Comment:
I think this should still be return amount *minus* fees as this is different than the outcomeSharesAmount calculation above

Reply:
Please also update the sell action on the FPMM level